orchestration/dagster_orchestration/hca_manage/dataset.py

```diff
@@ -51,3 +54,9 @@ def create_dataset(args: argparse.Namespace, host: str) -> JobId:
         billing_profile_id=args.billing_profile_id,
         schema_path=args.schema_path
     )
+
+
+def query_dataset(args: argparse.Namespace, host: str) -> str:
+    hca = HcaManage(environment=args.env, data_repo_client=get_api_client(host=host), dataset=args.dataset_name)
+    print(hca.enumerate_dataset())
+    return hca.enumerate_dataset()
```

Comment:
🔧 This function should not have a return and have a return type annotation  `None`; it's redundant to return the dataset payload as nothing is consuming it.

Reply:
ok makes sense, will change